src/sprintFour/RecursiveFizzBuzz.java

```diff
@@ -0,0 +1,23 @@
+package sprintFour;
+
+class RecursiveFizzBuzz {
+    private static void evaluate(int a, int b) {
```

Comment:
it would be better to pass only one input value evaluate(100)

Reply:
My program accepts two integers a and b and it appears to be working correctly. Would you like me to redo this assignment with one input?